lib/elements/dropDown.js

```diff
@@ -30,66 +34,164 @@ class DropDown extends Element {
+        return {
+          error: 'Cannot select multiple values on a single select dropdown',
+          stack: '',
+        };
       }
 
-      if (this.options.length > value.index) {
-        let ele = selectAndDispatchEvent(this, value.index);
-        found_value['isAvailable'] = ele;
+      values.every((value) => {
+        if (this.options.length > value && value != null) {
+          if (this.options[value].disabled) {
+            found_value[value] = { isDisabled: true };
+            return false;
+          } else {
+            let ele = selectAndDispatchEvent(this, value);
+            found_value[value] = { isAvailable: ele };
+            return true;
+          }
+        } else if (!value) {
+          if (Object.keys(found_value).length <= 0) {
+            return false;
+          } else {
+            found_value[value] = { isAvailable: false };
+            return false;
+          }
+        }
+      });
+
+      if (Object.keys(found_value).length > 0) {
         return found_value;
       }
 
-      for (var i = 0; i < this.options.length; i++) {
-        let option = this.options[i];
-        if (option.text === value || option.value === value) {
-          if (option.disabled) {
-            found_value['isDisabled'] = true;
-            break;
+      values.every((value) => {
+        if (!value) {
+          if (Object.keys(found_value).length <= 0) {
+            return false;
+          } else {
+            found_value[value] = { isAvailable: false };
+            return false;
           }
-          let ele = selectAndDispatchEvent(this, i);
-          found_value['isAvailable'] = ele;
-          break;
         }
-      }
+        let found = false;
+        let disabled = false;
+        for (let i = 0; i < this.options.length; i++) {
+          let option = this.options[i];
+          if (option.text === value || option.value === value) {
+            found = true;
+            if (option.disabled) {
+              found_value[value] = { isDisabled: true };
+              disabled = true;
+              break;
+            } else {
+              let ele = selectAndDispatchEvent(this, i);
+              found_value[value] = { isAvailable: ele };
+              return true;
+            }
+          }
+        }
+        if (disabled) {
+          return false;
+        }
+        if (!found) {
+          found_value[value] = { isAvailable: false };
+          return false;
+        }
+      });
       return found_value;
     }
     const options = setNavigationOptions({});
 
     // RegExp is not serialised. So loop through dropdown text and find a match
     // before invoking via CDP API
-    let valueToSelect = value;
-    if (isRegex(value)) {
+
+    let valuesToSelect = values;
+    if (Array.isArray(values)) {
+      //do nothing
+    } else if (typeof values == 'undefined') {
+      valuesToSelect = [undefined];
+    } else if (!values) {
+      valuesToSelect = [null];
```

Comment:
What is the need to set null and undefined?

Reply:
Fixed.